### PR TITLE
Add New Zealand shields

### LINKED ISSUE #379: New Zealand State Highway Shields
Proposed is to add support for New Zealand State Highways:
tagging: `network=NZ:SH`
wikipedia: https://en.wikipedia.org/wiki/New_Zealand_state_highway_network
graphics: image

### PR DESCRIPTION
**Pull request by claysmalley**
Fixes #379.

Screenshot from 2022-06-25 18-18-58 Screenshot from 2022-06-25 18-19-23

### CODE REVIEW

File: src/js/shield_defs.js [hunk 1]
```diff
@@ -2997,7 +3065,22 @@ export function loadShields(shieldImages) {
   // Kosovo
   shields["XK:motorway"] = wideHexagonGreenShield;
 
-  //Ref-specific cases. Additional entries should be documented in CONTRIBUTE.md
+  // OCEANIA
+
+  // New Zealand
+  shields["NZ:SH"] = {
+    backgroundImage: shieldImages.shield40_nz_sh,
```
Comment (1ec5) [inline src/js/shield_defs.js:3072]: There are some three-digit routes too. I assume you based this image on the Interstate or Business Interstate shield, so it should be pretty straightforward to make a three-digit one too?

**Review by 1ec5 — COMMENTED** [review]
Hah, you beat me to it! Remember to rerun `npm run status_map` to add the first southern hemisphere country to the shield coverage map!

**Comment by 1ec5** [comment]
Here are the SVGs I had stashed away:

shield40_nz_state_2 shield40_nz_state_3

File: src/js/shield_defs.js (hunk 1 above)
Comment (claysmalley) [reply src/js/shield_defs.js]: I did indeed base it on the Interstate shield :smile:

A 3-digit variant has been added:

Screenshot from 2022-06-25 21-39-32 Screenshot from 2022-06-25 21-39-54

**Review by 1ec5 — APPROVED** [review]

**Pull request merged**